### PR TITLE
[WIP] Add creation of pull request for repository permissions updater

### PR DESCRIPTION
**Pull request by slide**
This will create a PR in the repository-permissions-updater for the item that is being hosted. This is a WIP because the fields in JIRA have not been created yet.

### CODE REVIEW

**Comment by slide** [comment]
@daniel-beck, @oleg-nenashev, @olblak, @batmat please take a look and let me know what you think. I tried to create a test for this, but there are some classes that I had a hard time mocking correctly from the Github library.

**Comment by slide** [comment]
It creates a PR like this: https://github.com/slide-testing/repository-permissions-updater/pull/2

File: src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java [hunk 1]
```diff
@@ -369,14 +382,22 @@ private void setupHosting(Channel channel, User sender, String hostingId, String
...
+            for (String u : githubUserList.split("\\n")) {
                 if(StringUtils.isNotBlank(u))
-                    users.add(u.trim());
+                    githubUsers.add(u.trim());
             }
             String forkTo = JiraHelper.getFieldValueOrDefault(issue, FORK_TO_JIRA_FIELD, defaultForkTo);
 
-            if(StringUtils.isBlank(forkFrom) || StringUtils.isBlank(forkTo) || users.isEmpty()) {
+            String releaseUserList = JiraHelper.getFieldValueOrDefault(issue, RELEASE_USER_LIST_JIRA_FIELD, "");
+            for (String u : releaseUserList.split("\\n")) {
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java]: While this does not exist, a sane default is the user name who filed the HOSTING issue.

Comment (slide) [reply src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java]: I totally missed your feedback. I need to get back to this.

**Pull request closed without merging**